content/chainguard/administration/how-to-install-chainctl.md

```diff
@@ -71,6 +71,23 @@ sudo install -o $UID -g $GID -m 0755 chainctl /usr/local/bin/
 
 At this point, you'll be able to use the `chainctl` command.
 
+### Installing with `curl` in Windows Powershell
```

Comment:
I feel like if I were a Windows user I would have attempted the steps in the section before this because it said it was multiplatform. It might be worth placing a very short note there or at the start alerting Windows user to this section and/or linking to it.

Reply:
I think that's fair. Will add this in.